src/pages/Main.tsx

```diff
@@ -2,43 +2,62 @@ import Badges from 'components/badges/Badges'
 import CardSeparator from 'components/CardSeparator'
 import ProofsCard from 'components/proofs/Card'
 import ZkProofHint from 'components/ZkProofHint'
+import ZkProofHintWhenLg from 'components/proofs/ZkProofHintWhenLg'
 import classnames, {
   alignItems,
   display,
   flexDirection,
+  inset,
   justifyContent,
+  margin,
+  maxWidth,
+  position,
+  width,
 } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 
-const mainBlock = classnames(
+const cardsContainer = classnames(
   display('flex'),
   flexDirection('flex-col', 'lg:flex-row'),
   alignItems('items-center', 'lg:items-stretch'),
-  justifyContent('lg:justify-center')
+  justifyContent('lg:justify-center'),
+  margin('ml-10', 'mr-10')
+)
+
+const mainContainer = classnames(width('w-full'), maxWidth('max-w-1244'))
+
+const proofContainer = classnames(
+  position('lg:relative'),
+  inset('lg:right-1/4')
 )
 
 export default function () {
   const { lg } = useBreakpoints()
 
   return (
-    <div className={mainBlock}>
-      <ProofsCard />
-      <CardSeparator
-        numberOfLines={3}
-        gradient="accent-to-secondary"
-        vertical={!lg}
-      />
-      <Badges />
-      {!lg && (
-        <>
-          <CardSeparator
-            numberOfLines={1}
-            gradient="secondary-to-transparent"
-            vertical
-          />
-          <ZkProofHint />
-        </>
-      )}
+    <div className={mainContainer}>
```

Comment:
Do we really need to nest two `div`s into each other like this? Usually you can avoid doing it

Reply:
This mainBlock div holds two separate divs, the cardsContainer for positioning the cards, and then the proofHintContainer for positioning the ZKProofHint

Reply:
Yeah, I can see it now, thanks